### ISSUE TITLE
rank by loss

### ISSUE THREAD

**Issue by ebgoldstein** [ebgoldstein/SegmentationActiveLearning#4]
call model.evaluate on images and use `reduction= None` in the loss

https://keras.io/api/losses/

**Comment by ebgoldstein** [comment]
High Loss examples will be illustrative..
One idea is to find images w/ high loss, and then find similar images and label those..  
this can be done via the embeddings from the autoencoder.
